packages/common/src/compiler/translator.ts

```diff
@@ -140,6 +140,15 @@ const convertDbtMetricToLightdashMetric = (
             `Cannot determine model from dbt metric "${metric.name}" metric.model is "${metric.model}" but should be "ref('my_model')"`,
         );
     }
+    let sql = defaultSql(metric.name);
```

Comment:
does this just return an empty string if metric.name is undefined?

Reply:
metric name is required